app/build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
     debugImplementation "com.squareup.leakcanary:leakcanary-android:$rootProject.leakCanaryVersion"
     releaseImplementation "com.squareup.leakcanary:leakcanary-android-no-op:$rootProject.leakCanaryVersion"
     testImplementation "com.squareup.leakcanary:leakcanary-android-no-op:$rootProject.leakCanaryVersion"
+    implementation 'com.github.anastr:speedviewlib:1.2.0'
```

Comment:
Please add this line above the dependencies of Leak Canary so as to keep all dependencies other than leak canary together. Also try to use keep version in project level gradle file rather than hardcoding it here.